README.md

```
@@ -1,45 +1,362 @@
-3. Run the MCP server:
+3. **Run the MCP server:**
 ```bash
 uv run python main.py
 ```
 
-## Configuration
+## ⚙️ Configuration
 
-Set your Elasticsearch host in `.env`:
-```
+### Environment Variables (`.env`)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)